bgcval2/download_from_mass.py

```diff
@@ -409,15 +409,18 @@ def download_from_mass(
     outputFold = folder([paths.ModelFolder_pref,  jobID,] )
     # make this folder group writeable. 
     st = os.stat(outputFold)
-    os.chmod(outputFold, st.st_mode | stat.S_IWGRP)
+
+    try: os.chmod(outputFold, st.st_mode | stat.S_IWGRP)
+    except: pass
```

Comment:
This should be something, not except pass.

Reply:
fully agreed, maybe an `OSError: could not change permissions on such and such`?